Src/BootCamp.Chapter/MakeStuffWithInput.cs

```diff
@@ -0,0 +1,244 @@
+        public static string MakefullstringRichestPeople(int numberOfPeople, string people, decimal amount)
+        {
+            if (numberOfPeople > 1)
+            {
+                return people + " are the richest people. ¤" + amount + ".";
+            }
+            else
+            {
+                return people + " is the richest person. ¤" + amount + ".";
+            }
+        }
+        /// <summary>
+        /// This Gives back a string that reads: people have the least money. -¤=amount.
+        /// </summary>
+        /// <param name="numberOfPeople"></param>
+        /// <param name="people">string of person or persons</param>
+        /// <param name="amount">the amount of money</param>
+        /// <returns></returns>
+        public static string MakefullstringPoorestPeople(int numberOfPeople, string people, decimal amount)
+        {
+            if (numberOfPeople > 1)
+            {
+                if (amount < 0)
+                {
+                    return people + " have the least money. -¤" + Math.Abs(amount) + ".";
+                }
+                else
+                {
+                    return people + " have the least money. ¤" + amount + ".";
+                }
+
+            }
+            else
+            {
+                if (amount < 0)
+                {
+                    return people + " has the least money. -¤" + Math.Abs(amount) + ".";
+                }
+                else
+                {
+                    return people + " has the least money. ¤" + amount + ".";
+                }
+
+            }
+        }
+        /// <summary>
+        /// Gives back a string that reads: people lost the most money. ¤ amount.
+        /// </summary>
+        /// <param name="people">string of person or persons</param>
+        /// <param name="amount">the amount of money</param>
+        /// <returns></returns>
+        public static string MakefullstringBiggestLossPeople(string people, decimal amount)
+        {
+            if (people == "")
+            {
+                return "N/A.";
+            }
+            if (amount < 0)
+            {
+                return people + " lost the most money. -¤" + Math.Abs(amount) + ".";
+            }
+            else
+            {
+                return people + " lost the most money. ¤" + amount + ".";
+            }
+
+        }
+        /// <summary>
+        /// Gives back a string that reads: people had the most money ever. ¤ amount.
+        /// </summary>
+        /// <param name="numberOfPeople"></param>
+        /// <param name="people">string of person or persons</param>
+        /// <param name="amount">the amount of money</param>
+        /// <returns></returns>
+        public static string MakefullstringHighestBalanceEver(int numberOfPeople, string people, decimal amount)
+        {
+            if (people == "")
+            {
+                return "N/A.";
+            }
+            if (numberOfPeople > 1)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this conditional structure or edit its code blocks so that they're not all the same.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5123803)